jQuery/jquery.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
what is the use of this file

Reply:
The same html file got  added two times, I'll remove it.